src/Controls/tests/Xaml.UnitTests/CompiledTypeConverter.xaml.cs

```diff
@@ -41,6 +47,9 @@ public void CompiledTypeConverterAreInvoked(bool useCompiledXaml)
 				Assert.AreEqual(new Rect(0, 1, 2, 4), p.RectangleP);
 				Assert.AreEqual(new Rect(4, 8, 16, 32), p.RectangleBP);
 				Assert.AreEqual(new Point(1, 2), p.PointP);
+				Assert.AreEqual(Brush.Red, p.BrushByName);
+				Assert.AreEqual(new Color(1, 2, 3, 0), ((SolidColorBrush)p.BrushByARGB).Color);
+				Assert.AreEqual(new Color(1, 2, 3), ((SolidColorBrush)p.BrushByRGB).Color);
```

Comment:
I also sometimes prevent regression in this kind of code by, in the unit test, walk the IL and make sure  we do not invoke the original converter (see Xaml.UnitTests/FontSize.xaml.cs)